slave-base/Dockerfile.localdev

```diff
@@ -2,14 +2,16 @@ FROM quay.io/openshift/origin-cli
 
 MAINTAINER Akram Ben Aissi <abenaiss@redhat.com>
 
-ENV HOME=/home/jenkins
+ENV HOME=/home/jenkins \
+    PROMPT_COMMAND="./usr/local/bin/scl_enable"
```

Comment:
I didn't get why we need PROMPT_COMMAND yet ?
if we need to enable scl_enable on container start, this should be made using s2i/run command

Reply:
removed prompt command